client/src/components/DataEntry/Ballot.tsx

```diff
@@ -101,7 +101,13 @@ const Ballot: React.FC<IProps> = ({
       <BallotRow>
         <div className="ballot-side">
           <H4>Current ballot:</H4>
-          <div>Tabulator: {ballot.batch.tabulator}</div>
+          {ballot.batch.tabulator && (
+            <div>Tabulator: {ballot.batch.tabulator}</div>
+          )}
+          {ballot.batch.container && (
+            <div>Container: {ballot.batch.container}</div>
+          )}
```

Comment:
Container should go before tabulator (the container is the box that multiple tabulator batches might be in, so it's a higher level of hierarchy)